process/peer/process.go

```diff
@@ -730,6 +733,32 @@ func (vs *validatorStatistics) getTempRating(s string) uint32 {
 	return peer.GetTempRating()
 }
 
+func (vs *validatorStatistics) display(s string) {
```

Comment:
maybe change s to blsPubKey, or accountKey

Reply:
changed to validatorKey